src/app/components/Footer/List/index.tsx

```diff
@@ -0,0 +1,44 @@
+/** @jsx jsx */
+import { jsx } from '@emotion/react';
+import { FooterLink } from '#app/models/types/serviceConfig';
+import Link from '../Link';
+import styles, { gridTemplateRows } from './index.styles';
+
+export default ({
+  elements = [],
+  trustProjectLink,
+}: {
+  elements?: (jsx.JSX.Element | null)[];
```

Comment:
I'd maybe use `ReactElement` or `ReactNode` here. `ReactElement` will be an actual DOM element, whereas `ReactNode` can be anything that React can render, like a DOM element, string etc.

Reply:
I think we did try `ReactElement` and had TS errors, but can give`ReactNode` a shot to see if it works? This was a bit of a tricky one, so if we get stuck, we might need to call upon you for help @amoore108!

Reply:
Changed it to ReactElement and it works! We had loads of TS issues at the time and it didn't work but now we have fixed those, this is working! Thanks